pkg/ensurance/analyzer/analyzer.go

```diff
@@ -504,8 +503,14 @@ func combineEvictDuplicate(e *executor.EvictExecutor, evictPods executor.EvictPo
 		if i := e.EvictPods.Find(ep.PodKey); i == -1 {
 			e.EvictPods = append(e.EvictPods, ep)
 		} else {
-			if ep.DeletionGracePeriodSeconds < e.EvictPods[i].DeletionGracePeriodSeconds {
-				e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
+			if ep.DeletionGracePeriodSeconds != nil {
+				if e.EvictPods[i].DeletionGracePeriodSeconds == nil {
+					e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
+				} else {
+					if *(e.EvictPods[i].DeletionGracePeriodSeconds) > *(ep.DeletionGracePeriodSeconds) {
```

Comment:
maybe a little duplicated
```
if e.EvictPods[i].DeletionGracePeriodSeconds == nil || if *(e.EvictPods[i].DeletionGracePeriodSeconds) > *(ep.DeletionGracePeriodSeconds){
    e.EvictPods[i].DeletionGracePeriodSeconds = ep.DeletionGracePeriodSeconds
}
```

Reply:
if the ep.DeletionGracePeriodSeconds is nil, it will crash

Reply:
done~